### DISCUSSION TITLE
A4988 Stepper HOWTO "Goto Position"?

### DISCUSSION THREAD

**Discussion by dougie996** [arendst/Tasmota#17453] [Q&A]
Hi all, 

I've been busy with Tasmota since a while, but got stuck at a special task. 

I have set up an ESP8266 with an A4988 Stepper Controller -> Working fine
I set Tasmota with SetOption80 1 and Stepper mode 4 -> Working fine

What do I want to achieve?

I whish to send commands fom another computer to Tasmota to move a number of steps into one or the other direction OR to drive to a certain step position.  The Stepper will drive a rotary table via a transmission reduction in e.g. 5° increments

Perfect would be to execute a command "turn Turntable 5° CW" or "goto 45°" by telling Tasmota how many steps into which direction. 

Is there an existing solution or how can I adapt to make this possible?

BR
Ralf

**Comment by TrDA-hab** [comment]
> Is there an existing solution or how can I adapt to make this possible?
> 
Yes, there is a solution:
https://tasmota.github.io/docs/A4988-Stepper-Motor-Controller/

**Comment by dougie996** [comment]
Thanks so much for taking care and the response, but this describes the hardware setup only - which I already got perfectly working.

What I'm looking for is the way How I can move a given numer of steps and how to combine that in a web interface command.

When I'm controlling the current setup via the web page slider, I see three commands in the console output:

1. Set the Target Position
2. Set Power2 on or off (which is direction)
3. Set Power1 On until Target position is reached, then turn Power 1 Off (Which is enable to movement)

Currently I have no clue how to combine that into a Web Command like http://<ip>/cm?cmnd= ???
